Assets/Scripts/Game/DaggerfallUI.cs

```diff
@@ -1309,6 +1311,68 @@ public DaggerfallMessageBox CreateHealthStatusBox(IUserInterfaceWindow previous
             return healthBox;
         }
 
+        public DaggerfallMessageBox CreateBankingStatusBox(IUserInterfaceWindow previous = null)
```

Comment:
Why is this in here and not in a banking class?

Reply:
There's nothing related to GUI in banking class, would you really prefer that method to be there?

Reply:
There's an entire UI banking class, could go in there. I'm asking more than saying it's wrong.. was curious is all.

Reply:
That's an option... Their lifetimes do not match, but I think this method can be made static, so that's an option

Reply:
To be perfectly honest, in this case I used CreateHealthStatusBox() as an example.
But I agree that it adds bloat to DaggerfallUI; maybe CreateHealthStatusBox should also be put somewhere else...

Reply:
I moved the code as static methods into DaggerfallBankingWindow.
Not great, but most banking UI code is now in one place.